cli/dts/lib.deno.worker.d.ts

```diff
@@ -7,8 +7,9 @@
 /// <reference lib="deno.shared_globals" />
 /// <reference lib="esnext" />
 
-declare interface DedicatedWorkerGlobalScope {
-  self: DedicatedWorkerGlobalScope & typeof globalThis;
+declare class WorkerGlobalScope extends EventTarget {
```

Comment:
Looks like both `WorkerGlobalScope` and `DedicatedWorkerGlobalScope` exist, need to amend.